old_node_modules/.yarn-integrity

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Oops, it looks like this is in a folder called `old_node_modules`--did you commit that accidentally?

Reply:
removed!